llpc/tool/llpcPipelineBuilder.cpp

```diff
@@ -84,19 +84,26 @@ bool PipelineBuilder::isGraphicsPipeline() const {
 //
 // @returns : Result::Success on success, other status on failure.
 Result PipelineBuilder::build() {
-  BinaryData pipeline = {};
+  // Use vector of BinaryData for future pipeline type
```

Comment:
1. nit: this sentence should end in a `.`
2. I'd be more explicit and say something like:
```c++
`// Even though `build*Pipeline` produces a single `BinaryData` today, future pipeline types (e.g., raytracing) will result multiple outputs. We use a vector instead of a single variable to prepare for that.
```